### PR TITLE
Reorganize the docs site

### PR DESCRIPTION
**Pull request by deliciousmonster**
Rewrite run nio in the cloud so it’s not a workshop
New left nav organizes stuff
A few cosmetic things inside some pages
A couple typos
Updated requirements

### CODE REVIEW

File: docs/SUMMARY.md [hunk 1]
```diff
@@ -38,6 +44,20 @@
     * [Block Patterns](blocks/block-development/block-patterns.md)
     * [The nio Framework](blocks/block-development/framework.md)
     * [Mixins](blocks/block-development/mixins.md)
+* [User Interfaces](ui/README.md)
+  * [How to Build a UI with nio](ui/build-a-ui.md)
+
+---
+* Core Components
```
Comment (mattdodge) [inline docs/SUMMARY.md]: I would rename this to `nio Core` probably, not all of these are technically components

File: docs/SUMMARY.md [hunk 2]
```diff
@@ -52,31 +72,26 @@
   * [Command](cli/command.md)
   * [Clone Services](cli/clone-services.md)
   * [Shutdown](cli/shutdown.md)
-* [Hardware](hardware/README.md)
-  * [Supported Compute and Install](hardware/compute-install.md)
-  * [Kits and Sensors](hardware/kits-sensors.md)
+* [Communications](communications/README.md)
 * [Data](data/README.md)
```
Comment (mattdodge) [inline docs/SUMMARY.md]: I don't think Data really belongs in this core section. Maybe there's a new section called "using nio" or something similar that includes data, UIs, and the CLI

File: docs/SUMMARY.md [hunk 3]
```diff
@@ -1,18 +1,24 @@
...
+* [Install nio](installation/README.md)
+  * [On Windows](installation/windows.md)
+  * [Supported Systems](installation/supported-systems.md)
+* [Run nio](running-nio/README.md)
+  * [In a nio-Managed Cloud](running-nio/in-the-cloud.md)
   * [Locally](running-nio/locally.md)
-  * [Using Docker](running-nio/docker.md)
-  * [Long Running Instances](running-nio/long-running.md)
+  * [In a Docker Container](running-nio/docker.md)
+  * [For a Long Time](running-nio/long-running.md)
```
Comment (mattdodge) [inline docs/SUMMARY.md]: Maybe "For Long Durations"

Or is there a better phrase we could use? It's really more like "how to really run nio in any kind of real setting"

File: docs/running-nio/README.md [hunk 4]
```diff
@@ -1,10 +1,11 @@
...
+---
+**Run nio**
 
-* [Run in the cloud](/running-nio/in-the-cloud.md)
-* [Running Locally](/running-nio/locally.md)
-* [Running with Docker](/running-nio/docker.md)
-* [Long Running nio Instances](/running-nio/long-running.md)
+* [In a nio-Managed Cloud](/running-nio/in-the-cloud.md): we run nio for you, and you build cloud-based solutions
+* [Locally](/running-nio/locally.md): you run nio wherever you want, and use the System Designer to build and administer it
+* [In a Docker Container](/running-nio/docker.md): containerize nio, and run it using Kubernetes or something like it
```
Comment (mattdodge) [inline docs/running-nio/README.md]: `s/something like it/any other Docker orchestration tool`

File: docs/system-designer/README.md [hunk 5]
```diff
@@ -1,43 +1,41 @@
...
+* To enter the service context and view the contents of a service, click a service name.
+* Learn more about [instances](/instances) and [services](/services)
 
 ### Breadcrumb
+* As you navigate from system to service, the breadcrumb allows you to easily identify your location in the system.
 
-As you travel up and down the levels of your system, note that the breadcrumb changes allowing you to easily identify your location in the system.
-
-![breadcrumb hierarchy](/img/cloud/hierarchy.gif)
+<center>
```
Comment (mattdodge) [inline docs/system-designer/README.md]: Do we need `<center>` tags around every image? Maybe we should look into changing the style rules for the theme. @kyliedale @jennyknuth thoughts?

File: docs/running-nio/README.md [hunk 6]
```diff
@@ -1,10 +1,11 @@
-# Running nio
+# Run nio
 
-nio is a software product that should run throughout your IoT system. These docs will help you figure out how to run nio in whatever way you choose.
+nio facilitates the creation of distributed computing solutions across an almost unlimited array of platforms and devices. As such, there are lots of ways for you to run nio. We've outlined some of the most common implementations below.
```
Comment (kyliedale) [inline docs/running-nio/README.md]: Per the new branding rules, this should be "The nio Platform"

File: docs/running-nio/in-the-cloud.md [hunk 7]
```diff
@@ -1,5 +1,59 @@
-# nio in the Cloud
+# nio-managed Cloud Instance
+
+While the true power of nio is experienced when you have multiple [local installations](/running-nio/locally) communicating across a distributed network, we also offer the ability to spin up cloud instances through the **System Designer**.
```
Comment (kyliedale) [inline docs/running-nio/in-the-cloud.md]: "the nio **System Designer**"

File: docs/running-nio/in-the-cloud.md [hunk 8]
```diff
@@ -1,5 +1,59 @@
...
+
+A nio-managed Cloud Instance allows you to get up and running quickly, and to start building services right away (although our local installation process usually only takes 5 minutes!).
```
Comment (kyliedale) [inline docs/running-nio/in-the-cloud.md]: I don't think Cloud or Instance should be capitalized here?

File: docs/running-nio/in-the-cloud.md [hunk 9]
```diff
@@ -1,5 +1,59 @@
...
+
+>**[info] Prerequisites**
+>
+>* **a nio account**: [Sign up here](https://app.n.io/signup)
+
+---
+## Create A System
```
Comment (kyliedale) [inline docs/running-nio/in-the-cloud.md]: The rest of our headings are sentence case, so "Create system"

File: docs/running-nio/in-the-cloud.md [hunk 10]
```diff
@@ -1,5 +1,59 @@
...
+
+
+1. Complete the **create new system** window:
+  1. In the **system name** box, enter a name for your system.
+  1. Click **accept**.
+
+####&nbsp;
+
+---
+## Create a nio-managed Cloud Instance
```
Comment (kyliedale) [inline docs/running-nio/in-the-cloud.md]: No caps on Cloud or Instance here either 🙂

File: docs/SUMMARY.md (hunk 3 above)
Comment (deliciousmonster) [reply docs/SUMMARY.md]: I thought about "In Production"...

File: docs/SUMMARY.md (hunk 1 above)
Comment (deliciousmonster) [reply docs/SUMMARY.md]: nio Core it is.

File: docs/SUMMARY.md (hunk 2 above)
Comment (deliciousmonster) [reply docs/SUMMARY.md]: Building with nio

File: docs/running-nio/README.md (hunk 4 above)
Comment (deliciousmonster) [reply docs/running-nio/README.md]: I had that, but it made the line wrap... I edited it down a bit.

File: docs/system-designer/README.md (hunk 5 above)
Comment (deliciousmonster) [reply docs/system-designer/README.md]: I switched to a style tag for each image... as I work through them, I'm getting closer to something we could convert to a class... they're just all pretty different... I recommend slow playing it until everything is 100%, then going down that road.

File: docs/running-nio/README.md (hunk 6 above)
Comment (deliciousmonster) [reply docs/running-nio/README.md]: done

File: docs/running-nio/in-the-cloud.md (hunk 7 above)
Comment (deliciousmonster) [reply docs/running-nio/in-the-cloud.md]: done

File: docs/running-nio/in-the-cloud.md (hunk 8 above)
Comment (deliciousmonster) [reply docs/running-nio/in-the-cloud.md]: fair

File: docs/running-nio/in-the-cloud.md (hunk 10 above)
Comment (deliciousmonster) [reply docs/running-nio/in-the-cloud.md]: ugh

File: docs/running-nio/in-the-cloud.md (hunk 9 above)
Comment (deliciousmonster) [reply docs/running-nio/in-the-cloud.md]: I did not notice that.

File: docs/system-designer/README.md (hunk 5 above)
Comment (jennyknuth) [reply docs/system-designer/README.md]: I agree that we certainly don't need every image centered and I like the new look. I am happy to create a class or a custom gitbook block for image placement and standardization. @deliciousmonster I am curious about the deprecated html tags/attributes and if/how best to use them in markdown. Are you using these temporarily or do you find they are the way to go to overcome markdown's limitations?

**Pull request merged**